src/scripts/components/VideoListItem.js

```diff
@@ -72,19 +72,53 @@ const Bar = styled.div`
 class UploadListItem extends Component<Props, void> {
   constructor (props) {
     super(props)
+    this.state = {
+      uploadProgress: 0,
+      transcodingProgress: 0,
+      totalProgress: 0
+    }
     this.handleClick = this.handleClick.bind(this)
   }
 
   handleClick () {
     this.props.onClick(this.props.video.id)
   }
 
+  componentWillReceiveProps (nextProps: Props): void {
```

Comment:
this code is the same of `VideoForm.js`, this could be written once? by the way there is an issue, because when the file has been transcoded is correctly updated in `VideoForm.js` but not in `VideoListItem.js`

Reply:
ok can you create an issue for the issue? :-)

Reply:
(or fix it of course :-))